backend/LexBoxApi/Controllers/LoginController.cs

```diff
@@ -49,20 +49,25 @@ public async Task<ActionResult<LexAuthUser>> Login(LoginRequest loginRequest)
         return user;
     }
 
+    [HttpPost("refresh")]
```

Comment:
A POST with no content seems a bit weird, but it's possible that in the future we'll decide to add content here, so let's leave this as a POST.